examples/responsive-vis/responsive-vis-utils.js

```diff
@@ -0,0 +1,122 @@
+import {randomNormal} from 'd3-random';
+import {scaleLinear, scaleQuantize, scaleSqrt} from 'd3-scale';
+import {max, range} from 'd3-array';
+
+// create fake data
+export function createData(dataSize, barChart = true) {
+  const gen1 = randomNormal(70, 15);
+  const gen2 = randomNormal(50, 8);
+  const gen = (i) => i % 3 ? gen1() : gen2();
+  const data = new Array(dataSize);
+  let index = -1;
+
+  while (index++ < dataSize) {
+    if (barChart) {
+      data[index] = {
+        x: gen(index),
+        y: `Point-${index}`,
```

Comment:
`y` being the only changing part, maybe switching these two ifs for a ternary could be better